packages/flutter/test/material/bottom_app_bar_test.dart

```diff
@@ -71,15 +73,15 @@ void main() {
     await pump(FloatingActionButtonLocation.endDocked);
     await expectLater(
       find.byKey(key),
-      matchesGoldenFile('bottom_app_bar.custom_shape.1.png'),
+      matchesGoldenFile('bottom_app_bar.custom_shape.1.1.png'),
```

Comment:
I guess that these new golden files are just temporary? Once this PR landed, we should be able to either replace the old png with this new png, or delete the old png?

Reply:
We delete the old ones.

Reply:
The instructions are to append a number to the end of the name - unfortunately for this file, it already has a number at the end of its name.